packages/schematics/update/update/npm.ts

```diff
@@ -101,6 +102,9 @@ export function getNpmPackageJson(
               http: options[0],
               https: options[1],
             },
+            ssl: {
+              strict: options[2] === 'true',
```

Comment:
If it is undefined it should stay as undefined to pick up the default.

Reply:
undefined will cause strict: false. strict is default to true. The key should go there only if strict is false

```
if(options[2]==='false') {
  opts = {...opts, {strict: false}}
}
```

ssl stuff should be maximally explicit

Reply:
Actually, undefined will cause it to be true.   It is explicitly handled: https://github.com/npm/npm-registry-client/blob/latest/index.js#L32

Reply:
Addressed and updated